.clitest/functions/.test.setup

```diff
@@ -0,0 +1,22 @@
+#!/bin/zsh
+
+# initialize znap like a user would
+source $PWD/znap.zsh
+
+# works with BSD and GNU gmktemp
+T_TEMPDIR=${"$( mktemp -d -t t_znap.XXXXXXXX )":A}
+
+# set znap location
+hash -d znap=$T_TEMPDIR
+
+# save fpath
+fpath_=( $fpath )
+
+# mockgit
+git() {
+  if [[ $1 == --version ]]; then
+    print 2.8.0
+  else
+    print $@[-1]
```

Comment:
Can you still make this `print -r -- $[@]`? Otherwise, an argument from a future test could break this.